src/Network/Network/PrivateLinkService/GetAzurePrivateLinkServiceCommand.cs

```diff
@@ -75,7 +75,7 @@ public override void Execute()
             }
             else 
             {
-                IPage<PrivateLinkService> plsPage;
+                IPage<Microsoft.Azure.Management.Network.Models.PrivateLinkService> plsPage;
```

Comment:
nit: suggest to use **using namespace**

Reply:
using this one now.
`using NM = Microsoft.Azure.Management.Network.Models;`